### DISCUSSION TITLE
Unable to delete table column

### DISCUSSION THREAD

**Discussion by YariKoen** [AntlerVC/firetable#756] [Support / Q&A] [answered]
Hello, every time I want to delete column, its not removed. Column deletion should be only on Rowy level (not influenced by Firestore rules), afaik. I wanted to just update `key` of column, but it seems not possible (would be nice btw).

![rowy_column_bug](https://user-images.githubusercontent.com/47338217/185982674-52c689c0-1ecc-40c1-b8b6-a77fc3681e0a.gif)

**Comment by harinij** [comment, marked as answer]
Hi @YariKoen only admin users are users with access to the table config collection of rowy are able to make table structure changes. Hence the delete column does not reflect correctly. We have updated our UI in recent releases to ensure the option is only shown to relevant users.

If you still have this issue, please let us know otherwise closing it out for now.
